sdk/program/src/stake/program.rs

```diff
@@ -0,0 +1 @@
+crate::declare_id!("Stake11111111111111111111111111111111111111");
```

Comment:
Oh I think you can just put this in `sdk/program/src/stake/mod.rs`, like:
```
pub mod program {
    crate::declare_id!("Stake11111111111111111111111111111111111111");
};
```

Reply:
Will do